docs/pipelines.md

```diff
@@ -1013,6 +1046,22 @@ spec:
 
 For an end-to-end example, see [PipelineRun with WhenExpressions](../examples/v1beta1/pipelineruns/pipelinerun-with-when-expressions.yaml).
 
+#### `WhenExpressions` using `Aggregate Execution Status` of `Tasks` in `Finally Tasks`
+
+`WhenExpressions` in `Finally Tasks` can utilize
+[`Aggregate Execution Status` of `Tasks`](#using-aggregate-execution-status-of-all-tasks) as demonstrated:
+
+```yaml
+finally:
+      - name: notify-any-failure # executed only when one or more tasks fail
+        when:
+          - input: $(tasks.status)
+            operator: in
+            values: ["Failed"]
+        taskRef:
+          name: notify-failure
+```
+
```

Comment:
same as above on adding a link to an example

Reply:
😜  done 👍